models/repo.go

```diff
@@ -470,6 +470,12 @@ func MigrateRepository(u *User, name, desc string, private, mirror bool, url str
 		return repo, fmt.Errorf("create update hook: %v", err)
 	}
 
+	// Check if repository is empty.
+	_, _, err = com.ExecCmdDir(repoPath, "git", "rev-parse", "--verify", "--quiet", "HEAD")
```

Comment:
Thanks for following up! And what about stderr? Will there just be an error and no stdout or stderr if it's empty?